src/Tests/WB.Tests.Unit.Designer/QuestionnaireVerificationTests/when_verifying_empty_questionnaire.cs

```diff
@@ -12,7 +12,6 @@ internal class when_verifying_empty_questionnaire : QuestionnaireVerifierTestsCo
         [NUnit.Framework.OneTimeSetUp] public void context () {
             questionnaire = CreateQuestionnaireDocument();

```

Comment:
Could you please add one small positive test? Cover page with one question a is treated as valid one.

Reply:
added